src/helpers/issue.ts

```diff
@@ -99,11 +99,11 @@ export const getCommentsOfIssue = async (issue_number: number): Promise<Comment[
   return result;
 };
 
-export const getAllIssueComments = async (issue_number: number): Promise<Comment[] | null> => {
+export const getAllIssueComments = async (issue_number: number): Promise<Comment[]> => {
   const context = getBotContext();
   const payload = context.payload as Payload;
 
-  let result: Comment[] | null = [];
```

Comment:
These also seem obviously intentional. I believe that the API sometimes returns `null` @0xcodercrane

Reply:
Its getting handle properly here.

Reply:
https://github.com/ubiquity/ubiquibot/pull/344#issuecomment-1567720041